### ISSUE TITLE
Create a "used strings" artifact from the build process for use by the translation utility

### ISSUE THREAD

**Issue by jbphet** [phetsims/rosetta#384]
One of the steps that the translation utility (Rosetta) current executes while presenting the translation UI to a user is to obtain the HTML of the sim being translated and extract a list of used string keys.  Since the sims can be pretty big, this can be time consuming, and results in a lot of wasted time and bandwidth since so little of HTML file is actually used.  It would be great if we could either produce a build artifact that contains the list of the string keys used in the currently live version of the sim, or a service that is performed by the PhET website that produces such a list by extracting it from the HTML that is local to the website.  Either way, there would be a way for the translation utility to obtains *just* the list of string keys.

**Comment by jonathanolson** [comment]
Ready for MR, might be pending based on https://github.com/phetsims/phet-io/issues/1974 and https://github.com/phetsims/sun/issues/859#issuecomment-1843618441

**Comment by jonathanolson** [comment]
Maintenance complete! @jbphet there should be `english-string-map.json` in all production sims, e.g. https://phet.colorado.edu/sims/html/balancing-chemical-equations/1.2.22/english-string-map.json.

Let me know if you need any help with changing rosetta to use this, as it will enable https://github.com/phetsims/chipper/issues/1377.

**Comment by jbphet** [comment]
The changes to rosetta for using the "built strings" file have been implemented.  I believe the next step is to deploy the sims with the compressed strings.

@jonathanolson - back to you to do some deployment.  Let's test at least one sim that has been deployed in this manner before doing them all.  Bring me back in when we're ready for that.

**Comment by jonathanolson** [comment]
I deployed a new version of chains with it, should be up on production for testing, see https://phet.colorado.edu/sims/html/chains/latest/chains_all.html

**Comment by jbphet** [comment]
I tested using Chains, and all looks good.  Here is a summary of what I did:

- [x] Verify the artifact is there
- [x] Verify the HTML file contains the new, compressed format for the strings
- [x] Verify the stats for this sim show up on the live utility for the ab locale (this sim is only visible to PhET team members)
- [x] Verify that when the page is loaded a message appears in Rosetta's log about getting the English string map file
- [x] Make modifications to the strings shown in the UI and verify that they appear correctly when using the "Test" button
- [x] Press the "Publish" button with some mods in place, verify the commit to the chains string file in babel looks correct

I think we can call this complete.  @jonathanolson will still need to publish the other sims, but that will be tracked in a different issue.  Closing.
